Services/DataBaseService/BrokerConsumers/FindConsumer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using DataBaseService.Repositories.Interfaces;
+using DTO;
+using DTO.BrokerRequests;
+using MassTransit;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DataBaseService.BrokerConsumers
+{
+    public class FindConsumer : IConsumer<InternalFindUserRequest>
+    {
+        private readonly IUserRepository userRepository;
+
+        private UserCredential GetUserCredentialById(Guid Id)
```

Comment:
Is this id of user tables or user credential? As I understood, client send us user id

Reply:
Why private methods before public?

Reply:
> Why private methods before public?

Because it is a good practice

Reply:
> Is this id of user tables or user credential? As I understood, client send us user id

Good question

Reply:
I think, that client aws sending us the UserCredential Id. We should check this with those who write the client.

Reply:
]But we have decided to write public then private as I remember

Reply:
> ]But we have decided to write public then private as I remember

We have this one indeed, mb we should consider changing it? I also don't think that private methods should go after public as it's more logical to write private first, because IMHO they are expected to be used in public methods so when you open a file and review the class, it's good to see all needed implementations before getting to usage of them in public methods

Reply:
> I think, that client was sending us the UserCredential Id. We should check this with those who write the client.

All this about argument name, change it on userId or userCredentialId

Reply:
Changed